nym-api/src/node_describe_cache/mod.rs

```diff
@@ -209,23 +236,48 @@ impl CacheItemProvider for NodeDescriptionProvider {
     }
 
     async fn try_refresh(&self) -> Result<Self::Item, Self::Error> {
```

Comment:
I don't remember from the top of my head where this information is persisted, but isn't it the case that the current described gateways are served via `/gateways/described` meaning they'd now confusingly also include mixnodes?

Reply:
Will add an endpoint and return via Role filtering, will also add one for returning all